packages/victory-histogram/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "victory-histogram",
+  "version": "34.1.3",
+  "description": "Histogram Component for Victory",
+  "keywords": [
+    "data visualization",
+    "React",
+    "d3",
+    "charting"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/formidablelabs/victory.git"
+  },
+  "homepage": "https://formidable.com/open-source/victory",
+  "main": "lib/index.js",
+  "module": "es/index.js",
+  "jsnext:main": "es/index.js",
+  "author": "Formidable",
+  "license": "MIT",
+  "dependencies": {
+    "d3-array": "^2.4.0",
+    "lodash": "^4.17.15",
+    "prop-types": "^15.5.8",
+    "victory-core": "^34.1.3"
```

Comment:
`bar` and `path-helper-methods` should be moved to `victory-core/victory-primitives` so that `victory-histogram doesn't need to depend on `victory-bar`